AK/Assertions.h

```diff
@@ -10,7 +10,7 @@
 #    include <Kernel/Assertions.h>
 #else
 #    include <assert.h>
-#    ifndef NDEBUG
+#    if !defined(NDEBUG) && !defined(_WIN32) && !defined(_WIN64)
```

Comment:
`_WIN32` is defined on 64-bit too, so I think the `_WIN64` bit isn't needed.

Why not use `defined(AK_OS_WINDOWS)` though?

Why disable VERIFY on Windows at all? This should probably have a `// FIXME: Enable on Win once #someissue is figured out`?

Reply:
This header doesn't include AK/Platform.h (for whatever reason).

It's not disabled per se, rather it redirects to our custom ak_verification_failed implementation. I'm not sure why we use `assert` at all even, when it has the known ndebug behavior. 

The real issue is that `assert` in the windows CRT isn't noreturn on purpose, so that it can e.g ask you to hook up a debugger and continue execution from there.

Reply:
I added a note and a FIXME here, but kept the _WIN32 since I'm not sure what adding AK/Platform.h would do to this file.